docs/tables/aws_guardduty_publishing_destination.md

```diff
@@ -0,0 +1,100 @@
+# Table: aws_guardduty_publishing_destination
+
+AWS Guardduty Publishing Destination provides a resource to export the guard duty findings. This requires an existing GuardDuty Detector.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  detector_id,
+  destination_id,
+  arn,
+  destination_arn,
+  status
+from
+  aws_guardduty_publishing_destination;
+```
+
+### List unverified publishing destination
+
+```sql
+select
+  destination_id,
+  arn,
+  status
+from
+  aws_guardduty_publishing_destination
+where
+  status = 'PENDING_VERIFICATION';
+```
+
+###  List publishing destination which are not encrypted
+
+```sql
+select
+  destination_id,
+  kms_key_arn,
+  status,
+  destination_type
+from
+  aws_guardduty_publishing_destination
+where
+  kms_key_arn is null;
+```
+
+### Count publishing destination by type
+
+```sql
+select
+  destination_type,
+  count(destination_id)
+from
+  aws_guardduty_publishing_destination
+group by 
+  destination_type
+order by
+  count desc;
+```
+
+### Get S3 bucket policy for publishing destination
+
+```sql
+select
+  d.destination_id,
+  d.destination_arn,
+  d.destination_type,
+  p ->> 'Sid' as sid,
+  p ->> 'Action' as policy_action,
+  p ->> 'Effect' as effect,
+  p -> 'Principal' ->> 'Service' as principal_service
+from
+  aws_guardduty_publishing_destination as d,
+  aws_s3_bucket as s,
+  jsonb_array_elements(s.policy -> 'Statement') as p
+where
+  d.destination_type = 'S3'
+and
+  s.arn = d.destination_arn;
+```
+
+### Get KMS Key policy document associate with publishing destination
```

Comment:
```suggestion
### Get KMS key policies associated with publishing destinations
```

`Key` should not be capitalized, as resource names are not normally capitalized (but service names are)